scout/build/panel.py

```diff
@@ -42,7 +42,7 @@ def build_gene(gene_info: dict, adapter) -> dict:
 
     # Add boolean flags
     gene_obj.update(
-        {key: True for key in ["reduced_penetrance", "mosaicism"] if gene_info.get(key)}
+        {key: gene_info.get(key) for key in ["reduced_penetrance", "mosaicism"] if key in gene_info}
```

Comment:
Ah, yes. This can potentially be painful. I dunno, I think I would have played it safe and also added a check against `None` values, although they should in theory not be there now. But, we can always fix if it breaks.